app/services/platform-apps/index.ts

```diff
@@ -177,10 +236,18 @@ export class PlatformAppsService extends
       devPort: DEV_PORT,
       poppedOutSlots: []
     });
-    localStorage.setItem(this.localStorageKey, JSON.stringify({
-      appPath, appToken
-    }));
-    this.appLoad.next(this.getApp(id));
+  }
+
+  addApp(app: ILoadedApp) {
+    const { id, appPath, appToken } = app;
+    this.ADD_APP(app);
+    if (app.unpacked) {
+      // store app in local storage
+      localStorage.setItem(this.localStorageKey, JSON.stringify({
+        appPath, appToken
+      }));
+      this.appLoad.next(this.getApp(id));
```

Comment:
I think the `appLoad` event is relevant for all types of apps, unpacked and production alike.  Is there a reason you're only sending it for unpacked apps here?

Reply:
I was under the impression that we were only using appLoad to keep track of unpacked apps load/unload event, but that make sense to do it for all. thanks